.github/workflows/ci.yml

```
@@ -0,0 +1,37 @@
+        run: chmod +x ./gradlew
+
+      - name: Run Build and Test
+        run: ./gradlew clean build --no-daemon
+
+      - name: Login to DockerHub
+        run: echo "${{ secrets.DOCKER_PASSWORD }}" | docker login -u "${{ secrets.DOCKER_USERNAME }}" --password-stdin
+
+      - name: Build and push Docker image
+        run: |
+          docker build -t ${{ secrets.DOCKER_USERNAME }}/${{ secrets.DOCKER_REPO }}:latest .
+          docker push ${{ secrets.DOCKER_USERNAME }}/${{ secrets.DOCKER_REPO }}:latest
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)